### PR TITLE
H-1020: Add basic browser extension tests

### PR DESCRIPTION
**Pull request by CiaranMn**
## 🌟 What is the purpose of this PR?

Adds some basic tests for the browser extension.

### 🚢 Has this modified a publishable library?

This PR:

- [x] does not modify any publishable blocks or libraries, or modifications do not need publishing

### 📜 Does this require a change to the docs?

The changes in this PR:

- [x] are internal and do not require a docs change

### 🕸️ Does this require a change to the Turbo Graph?

The changes in this PR:

- [x] affected the execution graph, and the `turbo.json`'s have been updated to reflect this

## ⚠️ Known issues

This doesn't test that the correct API are calls for inference, which is blocked on being able to do this (or knowing how to do this) via Playwright – I've left a `@todo`.

## 🛡 What tests cover this?

- These ones.

## ❓ How to test this?

1. Check the test action

### CODE REVIEW

File: tests/hash-playwright/package.json
Comment (CiaranMn) [reply tests/hash-playwright/package.json:11]: Thanks!

File: tests/hash-playwright/tests/browser-plugin.spec.ts
Comment (benwerner01) [inline tests/hash-playwright/tests/browser-plugin.spec.ts:38]: Maybe there should be a check here to see whether the quick note input is present or something like that to confirm the browser extension is in the logged in state? If that's not something we want to do here, maybe the comment needs updating.

**Review by benwerner01 — APPROVED** [review]
Nice work Ciaran, looks good to me 👍 Had one minor suggestion, but don't think it should block merging.

Just to flag there's a few linting errors occurring in CI, so will re-approve once those are addressed.

File: tests/hash-playwright/tests/browser-plugin.spec.ts
Comment (CiaranMn) [reply tests/hash-playwright/tests/browser-plugin.spec.ts]: Yep fair, updated in https://github.com/hashintel/hash/pull/3894/commits/d42c5d017f847664fb1c242c850a327d8e703701

**Review by benwerner01 — DISMISSED** [review]

**Review by benwerner01 — APPROVED** [review]

**Pull request merged**
